src/oatpp/core/parser/Caret.cpp

```diff
@@ -375,6 +377,9 @@ v_int64 Caret::StateSaveGuard::getSavedErrorCode() {
   v_float32 Caret::parseFloat32(){
     char* end;
     char* start = (char*)&m_data[m_pos];
+
+    parser::json::Utils::convertSingleDecimalSeparatorFromJsonToLocale((p_char8) start);
```

Comment:
We can't add this here - since `Caret` is a general-purpose class.
It's not serving JSON only.